How-To-Guidelines.md

```diff
@@ -0,0 +1,30 @@
+# Howto Guidelines
+
+Simply create a howto under the [howto](https://docs.cacti.net/howto)
+namespace, separated by a colon. ie: `http://docs.cacti.net/howto:my_howto`
+
+If you use level one separation for your header,
+
+```markdown
+# My Howto
+```
+
+it will appear on the [howto](https://docs.cacti.net/howto) index page.
```

Comment:
This will need to be reviewed as I'm not sure it's correct.  We aren't generating ToC's like we used to so level 2/3 isn't the same thing now.